src/it/junit/custom-output-directory/verify.groovy

```diff
@@ -15,8 +18,8 @@ import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
 
 @RunWith(Cucumber.class)
-@CucumberOptions(strict = true, features = {"classpath:features/feature1.feature"}, plugin = {"json:target/my-custom-dir/1.json",
-"pretty"}, monochrome = false, tags = {"@complete", "@accepted"}, glue = { "foo", "bar" })
+@CucumberOptions(strict = true, features = {\"""" + feature1.absolutePath + """\"}, plugin = {"json:target/my-custom-dir/1.json"},
```

Comment:
It actually compiles just fine. Although, I'm new to Groovy and was unaware of the technique you suggested which is better because it's much cleaner. I'm updating all of these now and pushing in just a moment.